api/v2.0/swagger.yaml

```diff
@@ -9244,4 +9320,58 @@ definitions:
         type: array
         items:
           $ref: '#/definitions/ScanDataExportExecution'
-        description: The list of scan data export executions
\ No newline at end of file
+        description: The list of scan data export executions
+  WorkerPool:
+    type: object
+    description: the job pool
+    properties: 
+      pid:
+        type: integer
+        description: the id of the pool
+      worker_pool_id:
```

Comment:
what is the difference from `worker_pool_id` to `pid`. Description is not clear.

Reply:
Updated, thanks!